content/influxdb/cloud-dedicated/reference/release-notes/influxctl.md

```diff
@@ -0,0 +1,256 @@
+  should only accept retention policy updates as a flag.
+- Update [`influxctl token create`](/influxdb/cloud-dedicated/reference/cli/influxctl/token/create/)
+  and [`influxctl token update`](/influxdb/cloud-dedicated/reference/cli/influxctl/token/update/)
+  help information with examples that use multiple permission flags.
+- Update [`influxctl cluster get`](/influxdb/cloud-dedicated/reference/cli/influxctl/cluster/get/)
+  help text.
+- Switch email param ordering.
+
+### Dependency updates
+
+- Update `golang.org/x/mod` from 0.8.0 to 0.12.0.
+
+### Miscellaneous
+
+- Clean up log formatting.
+- Remove extra debug output of account ID.
+
+---
+
+## v2.0.1 {date="2023-08-15"}
+
+### Bug fixes
+
+- Return an error when using unrecognized TOML configuration options.
+
+---
+
+## v2.0.0 {date="2023-08-09"}
+
+### Features
+
+- Add support for both InfluxDB Cloud Dedicated and InfluxDB Clustered.
+- Provide public distributions through <https://portal.influxdata.com/downloads/>
+  and the <https://repos.influxdata.com/> repository.
+- The `influxctl` configuration file is now a single file that you can
+  optionally pass in via the CLI.
+- Add additional options to [`influxctl database`](/influxdb/cloud-dedicated/reference/cli/influxctl/database/)
+  and [`influxctl token`](/influxdb/cloud-dedicated/reference/cli/influxctl/token/)
+  subcommands.
+- Introduce [`influxctl cluster`](/influxdb/cloud-dedicated/reference/cli/influxctl/cluster/)
+  subcommands.
+- Remove the `influxctl init` subcommand to avoid additional complexity of an
+  InfluxDB Cloud Dedicated configuration.
+
+{{% note %}}
+### Migrate from influxctl 1.x to 2.0
+
+`influxctl` 2.0+ supports multiple InfluxDB v3 products.
+To simplify connection configuration management, all configurations are now managed
+in a single file rather than separate files for each connection configuration.
+
+To migration `influxctl` 1.x configuration files to the 2.x format, use the
+following guidelines:
+
+1.  Create a 2.0+ configuration file (`config.toml`) at the default location
+    for your operating system.
+    _See [Create a configuration file](/influxdb/cloud-dedicated/reference/cli/influxctl/#create-a-configuration-file)_.
+
+2.  Copy the `account_id` and `cluster_id` credentials from your `influxctl` 1.x
+    configuration file and add them to a `[[profile]]` TOML table along with the
+    following fields:
+
+    - **name**: Profile name
+    - **product**: InfluxDB product (`dedicated`)
+
+    For example, the following 1.x configuration file:
+
+    ```toml
+    account_id = "dff3ee52-b494-47c1-9e2c-ab59d90d94eb"
+    cluster_id = "5827cdeb-b868-4446-b40e-e08de116fddf"
+    ```
+
+    would become:
+
+    ```toml
+    [[profile]]
+        name = "default"
+        product = "dedicated"
+        account_id = "dff3ee52-b494-47c1-9e2c-ab59d90d94eb"
+        cluster_id = "5827cdeb-b868-4446-b40e-e08de116fddf"
+    ```
+{{% /note %}}
+
+### Features
```

Comment:
this appears to be the 2nd features section for the v2.0.0 release?

Reply:
Nice catch. I'll fix it.